src/NoteBook.js

```diff
@@ -0,0 +1,22 @@
+class Notebook {
+
+  constructor(listOfNotes) {
+    this.listOfNotes = ['I need to go and buy a football', 'I want to go bowling', 'go and code'];
+  
+  }
+
+  truncatedNotes(){
+    for (const noteIndex of this.listOfNotes) {
+      if(noteIndex.length > 20) { 
+        return noteIndex.substring(0, 20)+'...'
+      }else if(noteIndex.length <= 20) {
+        return noteIndex
+      }
+    }
+  }
+//note.content
+openNotes(note){
+  return this.listOfNotes[note];
```

Comment:
This will return undefined, when obtaining an element from an array using `[]` the argument inside the square brackets has to be an index of the element, (so an integer with 0 <= integer < `array.length`).

Currently, since you're passing the "note" through this `openNotes` function that means that they or the program already has the note object, so then they could just do `note.content` to view what's written, so I think it would make more sense to pass just an index `i` through this function and return `this.listOfNotes[i]`.

Will let you think about this in more depth but I'm thinking that in practice if someone is on the UI and they see a truncated note that they want to open, they'll click on that html element to open it. So this means the html element is in someway tied to that given note from the array of notes. Maybe that means the id of the html element could correspond to the index of that specific note from within the array. Which was why I think it makes more sense to pass something like an integer through the function.